requirements.txt

```diff
@@ -18,3 +18,4 @@ ujson==1.33
 pytest==3.3.1
 pytest-cov==2.5.1
 unittest2==1.1.0
+mock == 2.0.0
```

Comment:
Should be using this https://docs.python.org/3/library/unittest.mock.html instead of installing via pip.